frontend/src/text_resources.js

```diff
@@ -14,7 +14,52 @@ let LOCALIZE = new LocalizedStrings({
       title: "The Public Service Commission's Competency Assessment Tool",
       welcomeMsg: "Welcome to the Compotency Assessment Tool.",
       description:
-        "Code for Canada and PSC are working on this tool to help assess managerial canadiates for the federal public service. It is in the alpha phase which means it is currently going through product testing and is subject to change."
+        "Code for Canada and PSC are working on this tool to help assess managerial canadiates for the federal public service. It is in the alpha phase which means it is currently going through product testing and is subject to change.",
+      login: {
+        title: "LOG IN",
+        content: {
+          title: "Login",
+          description:
+            "An account is required to proceed further. To log in, enter your credentials below.",
+          inputs: {
+            emailTitle: "Email Address:",
+            emailPlaceholder: "john.smith@outlook.ca",
+            passwordTitle: "Password:",
+            passwordPlaceholder: "Password"
+          }
+        },
+        button: "Login"
+      },
+      createAccount: {
+        title: "CREATE AN ACCOUNT",
+        content: {
+          title: "Create an account",
+          description:
+            "An account is required to proceed further. To create an account, fill out the following.",
+          inputs: {
+            firstNameTitle: "First Name:",
+            firstNamePlaceholder: "John",
+            lastNameTitle: "Last Name:",
+            lastNamePlaceholder: "Smith",
+            emailTitle: "Email Address:",
+            emailPlaceholder: "john.smith@outlook.ca",
+            passwordTitle: "Password (must be between 5-15 characters):",
+            passwordPlaceholder: "Password",
+            passwordErrors: {
+              description: "Your password must satisfy the following:",
+              Bullet1: "At least one upper case",
```

Comment:
The names of the strings should be more descriptive of the of what they are. Instead of `Bullet1`, maybe `uppercase`?

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/152/commits/013f93506b747359bafe5c5049c1eec7587eb71d